staff/henry-novoa/app-template/contact.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <header>
+        <h1>Contact</h1>                 
+    </header>
+    <nav> 
```

Comment:
It's better to have each item in a list in your nav bar. so you would have  an "ul" and each anchor tag would be inside a "li" tag

Reply:
agree

Reply:
I'm on it